drivers/spi/spi_ll_stm32.h

```diff
@@ -90,7 +90,7 @@ static inline uint32_t ll_func_tx_is_empty(SPI_TypeDef *spi)
 #if defined(CONFIG_SOC_SERIES_STM32MP1X) || \
 	defined(CONFIG_SOC_SERIES_STM32H7X) || \
 	defined(CONFIG_SOC_SERIES_STM32U5X)
-	return LL_SPI_IsActiveFlag_TXP(spi);
+	return LL_SPI_IsActiveFlag_TXC(spi);
```

Comment:
I think this commit should be removed, and we should keep TXP:
TXP mean there is space in the FIFO,
and ll_func_tx_is_empty() is used in spi_stm32_shift_m() spi_stm32_shift_s() 
So as soon as there is space in FIFO we can transfer a new packet in the FIFO.
No need to wait for transfer complete.

Reply:
Agree, checking the TXP flag instead of TXC (removing the commit)